docs/contexts/lds-jws2020-v1.json

```diff
@@ -1,6 +1,21 @@
 {
   "@context": {
-    "JsonWebKey2020": "https://w3id.org/security#JsonWebKey2020",
+    "privateKeyJwk": {
```

Comment:
If possible it would be good to drop privateKeyJwk in my opinion

Reply:
please don't advocate for security by obscurity here.

Reply:
Not trying to advocate for it from a security perspective. Just trying to account for a proper separation of concerns and since the rest of the data being defined in here is generally publicly available I think it would be better to define it under it's own context so that public data has a separate context from private data.

Reply:
two things
1. do you agree the context here should be the same as https://w3c-ccg.github.io/lds-jws2020/contexts/v1/ 
2. do you agree that https://w3c-ccg.github.io/lds-jws2020/contexts/v1/ is authoritative

if yes to both, let's merge this and consider changes in subsequent PRs

Reply:
1. Yes I do, and I believe that the content in that context is incorrect and should be split out (subsequent PR is fine)
2. I don't agree with this quite exactly, but it's a nuance that's largely irrelevant here. the `http://w3id.org/security/suites/jws-2020/v1` is what I'd consider authoritative and since it redirects here to that file it's effectively the same.

In any case, my review is not a blocking review. Hence why I've approved the PR. I'm fine with splitting this out in subsequent PRs, and not sure why my review has been received as blocking and semi hostile.

Reply:
See my comments on your PR to the redirection service: https://github.com/perma-id/w3id.org/pull/2547